conf/upgrade/upgrade_ui.yaml

```diff
@@ -0,0 +1,3 @@
+---
+UPGRADE:
+  upgrade_ui: True
```

Comment:
I'm thinking about renaming the `upgrade_ui` to `ui_upgrade` to have it aligned with [`ui_deployment`](https://github.com/red-hat-storage/ocs-ci/blob/master/conf/ocsci/ui_deployment.yaml#L4), what do you thing?